enhancements/oc/day2-add-nodes.md

```diff
@@ -0,0 +1,269 @@
+  interfaces:
+   - macAddress: 00:00:00:00:00:00
+     name: eth0
+   - macAddress: 00:00:00:00:00:01
+     name: eth1
+  networkConfig:
+   interfaces:
+     - name: eth0
+       type: ethernet
+       state: up
+       mac-address: 00:00:00:00:00:00
+       ipv4:
+         enabled: true
+         address:
+           - ip: 192.168.122.2
+             prefix-length: 23
+         dhcp: false
+- hostname: extra-worker-2
+  rootDeviceHints:
+   deviceName: /dev/sda
+  interfaces:
+   - macAddress: 00:00:00:00:00:02
+     name: eth0
+  networkConfig:
+   interfaces:
+     - name: eth0
+       type: ethernet
+       state: up
+       mac-address: 00:00:00:00:00:02
+       ipv4:
+         enabled: true
+         address:
+           - ip: 192.168.122.3
+             prefix-length: 23
+         dhcp: false
+```
+
+### monitor-add-nodes command
+
+The format of the command will be the following:
+
+`oc -–kubeconfig=’’ adm monitor-add-nodes <IPs list>`
+
+Where:
+- `–kubeconfig=’’``. Path to the kubeconfig file (or KUBECONFIG if available)
+- `IPs list`` is a list of the nodes IP to be monitored
+
+The command will try to connect to each specified node to retrieve initially the results of the pre-flight validations, and then 
+the progress of the activity. In case of validation failures, the command will report them to the console to allow the user to 
+take the requested actions and troubleshoot the issues. In the future this command could be enhanced to automatically approve the node CSRs
+(identifying the correct CSR requires the Node IP address, which this command already has as an input).
+
+### Workflow Description
+The working environment must satisfy the initial following conditions:
+- `oc` tool is available
+- A direct connection to the target cluster is available
+- kubeconfig file is available
+
+The following steps illustrates the expected workflow:
+1. The cluster administrator prepares the `nodes-config.yaml` file listing all the nodes to be added, optionally with their initial network configuration
+2. The cluster administrator runs the `oc adm add-nodes-image` command to generate the ISO
+3. The cluster administrator boots every node previously listed using the freshly generated ISO
+4. The cluster administrator runs the `oc adm monitor-add-nodes` command to monitor the activity progress
+5. Once a node csr is pending, the cluster administrator runs the `oc adm certificate approve <csr-id>` to complete the process for the specified node
+
+### API Extensions
+N/A
+
+### Implementation Details/Notes/Constraints
+The implementation will largely reuse the same approach / code present in the openshift/installer repo for creating an ISO image 
+for the installation, with a number of exceptions that will be detailed in the subsequent paragraphs.
+
+#### Proposed architecture
+Every node booted with the generated ISO will be independent for adding itself to the cluster - without any need to specify a single rendezvous IP, 
```

Comment:
suggest removing the comma at the end here, as it makes it slightly ambiguous in my reading.

Reply:
Ack